data/campaigns/The_South_Guard/scenarios/06a_Tidings_Good_and_Ill.cfg

```diff
@@ -365,7 +365,7 @@
             message= _ "Ho! Me see elves. Elves tasty? Me take elves to eat."
         [/message]
         [message]
-            race=elf
+            speaker=second_unit
```

Comment:
While you're here, maybe add `female_message` (just exactly the same with female^ prefix), since presumably this can potentially be spoken by either a male or a female message. (I'm not entirely sure if it matters for a plural, but I'd say it doesn't hurt.)